pkg/operator/ceph/cluster/rbd/config.go

```diff
@@ -34,7 +34,7 @@ const (
 `
 )
 
-// daemonConfig for a single rbd mirror
+// daemonConfig for a single rbd-mirror
```

Comment:
Why is the dash needed in rbd-mirror in comments and log entries? I thought the dash would just be needed when it's used as the name of a k8s resource. It's fine either way, just curious if others see it more readable like this.

Reply:
It was intended as an exercise in using consistent terminology, but after reading your comment, I too got a little confused. `rbd-mirror` here I take to mean `rbd mirroring resource`. I don't think there really is such a thing as "an rbd mirror". Rather, there is "rbd mirror**ing**", "rbd-mirror daemons", and Rook has "rbd-mirror daemon deployments". Ceph docs use the phrase `rbd-mirror daemon` presumably because the binary is `rbd-mirror`. I think if we are talking about a daemon or deployment `rbd-mirror` is the term to use over `rbd mirror`, and we leave the `daemon` part as implied as we do with mon, osds, etc. And then when talking about the act of mirroring it is `rbd mirroring`.

Reply:
There is a 50% chance I'm more clear on this now. :)